newspaper/extractors.py

```diff
@@ -195,6 +199,16 @@ def parse_date_str(date_str):
             if datetime_obj:
                 return datetime_obj
 
+        JSONLD_TAGS = [
+            'datePublished'
+        ]
+        jsonlde = JsonLdExtractor()
+        jsonlddata = jsonlde.extract_items(doc)
```

Comment:
I expect yes.

But I think the json-ld is widely used nowadays. If we decide to integrate json-ld into newspaper, we will probably need to restructure the code and move some priority of these parsers up.